README.md

```diff
@@ -138,7 +138,8 @@ kubectl apply -f manifest.yml
 
 ```bash
 kubectl apply -f deploy/rbac-secretproviderclass.yaml # update the namespace of the secrets-store-csi-driver ServiceAccount
-kubectl apply -f deploy/csidriver.yaml
+kubectl apply -f deploy/csidriver.yaml # deploy this yaml for kubernetes version 1.15 only
+kubectl apply -f deploy/csidriver-1.16.yaml # deploy this yaml for kubernetes version 1.16+
```

Comment:
should we make the 1.16+ one as the default? and comment out the <1.16 one as optional if they are on <1.16?

Reply:
@ritazh Are you suggesting we add the `volumeLifecycleMode` to `csidriver.yaml` and create a different one for 1.15 only (`csidriver-1.15.yaml`)?

Reply:
yes

Reply:
done!